astropy/io/ascii/tdat.py

```diff
@@ -388,6 +481,11 @@ def write(self, lines):
             "table_security",
         ]
         table_keywords = [kw for kw in table_keywords if kw in keywords.keys()]
+        for kw in ["record_delimiter", "field_delimiter"]:
+            val = keywords.pop(kw, None)
+            if val is not None:
+                warn(TdatFormatWarning(f"Skipping deprecated keyword: {kw}."))
```

Comment:
Oh, these are deprecated? Well, I guess you can ignore some of my previous comments.